core/GlobalState.cc

```diff
@@ -1052,14 +1056,14 @@ SymbolRef GlobalState::enterTypeArgument(Loc loc, SymbolRef owner, NameRef name,
     DEBUG_ONLY(categoryCounterInc("symbols", "type_argument"));
     wasModified_ = true;
 
-    owner.dataAllowingNone(*this)->typeArguments().emplace_back(result);
+    core::SymbolRef(*this, core::SymbolRef::Kind::Method, owner.id())
+        .dataAllowingNone(*this)
```

Comment:
I believe this is the only potential callsite of a `dataAllowingNone` method on `MethodRef`, so I didn't add it. 🤷